main/utils/errors.js

```diff
@@ -11,6 +10,7 @@ const showError = async (error, {title, reportToSentry} = {}) => {
 
   console.error(error);
   if (reportToSentry) {
+    const Sentry = require('./sentry');
```

Comment:
This was moved here to avoid a circular dependency

Reply:
Can you add a code comment about that so it's not accidentally moved back in the future?